docs/plugins/development/tables.md

```diff
@@ -87,3 +87,27 @@ The table column classes listed below are supported for use in plugins. These cl
     options:
       members:
         - __init__
+
+## Extending Core Tables
+
+!!! info "This feature was introduced in NetBox v3.7."
+
+Plugins can register their own custom columns on core tables using the `register_table_column()` utility function. This allows a plugin to attach additional information, such as relationships to its own models, to built-in object lists.
+
+```python
+import django_tables2
+
+from dcim.tables import SiteTable
+from utilities.tables import register_table_column
+
+mycol = django_tables2.Column(
```

Comment:
Just a question, now for best practices do we want to move people to making localizable plugins?  So update this to use the _( localized strings?

Reply:
Yeah, that's a good call-out.